gems/activerecord/6.0/activerecord.rbs

```diff
@@ -296,6 +296,8 @@ module ActiveRecord
   end
 
   module QueryMethods
+    type includee = String | Symbol | Array[includee] | Hash[untyped, untyped]
```

Comment:
I'm not sure this is the best place for this type alias. I'd really appreciate if you give me your opinion.
Thanks,

Reply:
I don't have a strong opinion of accuracy. So I'm okay to stop using recurrent definitions like this.

```suggestion
    type includee = String | Symbol | Array[untyped] | Hash[untyped, untyped]
```

Reply:
Finally, I removed `includee` type alias. It's only needed for recurrent definitions.